### PR TITLE
DTPools improvements

### PR DESCRIPTION
**Pull request by pavanbalaji**

### CODE REVIEW

**Comment by pavanbalaji** [comment]
test:jenkins/ch3/most
test:jenkins/ch4/most

**Comment by pavanbalaji** [comment]
test:jenkins/ch3/most
test:jenkins/ch4/most

File: test/mpi/rma/getfence1.c [hunk 1]
```diff
@@ -19,22 +19,22 @@ static char MTEST_Descrip[] = "Get with Fence";
 #define MAX_COUNT_SIZE (16000000)
 #define MAX_TYPE_SIZE  (16)
 
+int count;
```
Comment (gcongiu) [inline test/mpi/rma/getfence1.c]: Where is `count` initialized?

File: test/mpi/rma/putfence1.c [hunk 2]
```diff
@@ -19,22 +19,22 @@ static char MTEST_Descrip[] = "Put with Fence";
 #define MAX_COUNT_SIZE (16000000)
 #define MAX_TYPE_SIZE  (16)
 
+int count;
```
Comment (gcongiu) [inline test/mpi/rma/putfence1.c]: Same as above

File: test/mpi/rma/getfence1.c (hunk 1 above)
Comment (pavanbalaji) [reply test/mpi/rma/getfence1.c]: In the main program.

**Pull request closed without merging**